README.md

```diff
@@ -69,6 +69,7 @@ Find these templates in the `project_templates/` directory:
 - [nbreport](project_templates/nbreport/)
 - [stack_package](project_templates/stack_package/)
 - [technote_latex](project_templates/technote_latex/)
+- [technote_aastex](project_templates/technote_aastex/)
```

Comment:
Bump this above `technote_latex` to keep the alphabetical order.

Reply:
ok